extension/src/popup/views/Account/index.tsx

```diff
@@ -175,12 +187,6 @@ export const Account = () => {
           />
         )}
       </div>
-      <div>
-        {/* TODO - handle click */}
-        <Button fullWidth variant={Button.variant.tertiary}>
-          Manage Assets
-        </Button>
```

Comment:
Should we be removing this?

Reply:
My understanding is we're still adding the Trustlines module, unless I missed this

Reply:
I kept it but just moved it a couple lines up 😄  (to be in the `isFunded` view since it was showing up in the `nonFunded` view)

Reply:
Ah, lol - my bad. 👍